tests/report/display/test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+        rlRun "set -o pipefail"
+        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+        rlRun "run_dir=$tmp/original"
```

Comment:
The `run_dir` variable does not seems to be used. A copy & paste typo?

Reply:
Fixed in 8d37bce3.